Minor updates to excel-tutorial.md (#2044)
OfficeDev/office-js-docs-pr: docs/tutorials/excel-tutorial.md

6. Within the `sortTable()` function, replace `TODO1` with the following code. Note:

   - The code creates an array of `SortField` objects, which has just one member since the add-in only sorts on the Merchant column.

>    - The `key` property of a `SortField` object is the zero-based index of the column to sort.

Reviewer: I think we should be explicit about the sorting, since it's not just the column being sorted, but the entire table.

```suggestion
   - The `key` property of a `SortField` object is the zero-based index of the column used for sorting. The rows of the table are sorted based on the values in the referenced column.
```

   - The `Worksheet.freezePanes` collection is a set of panes in the worksheet that are pinned, or frozen, in place when the worksheet is scrolled.

>    - The `freezeRows` method takes as a parameter the number of rows, from the top, that are to be pinned in place. We pass `1` to pin the first row in place.

Reviewer: This is probably a separate issue, but "we" is typically avoided in our tutorials. I'll tag @ElizabethSamuel-MSFT to confirm.

Reviewer 2: Agreed. I think updating the sentence to "Pass `1` to pin the first row in place." should work.

Reviewer: Cool. Since there's a few more instances of "we" in this tutorial, let's make changing that a separate PR. Might be worth doing a search for "we " in the other tutorials too. 

BTW, "we" is okay in the context of a Microsoft recommendation (e.g., "We recommend using VS Code."

Author: Ok, thanks for the context! Happy to make a separate PR and remove "we" from the tutorial.

   > [!NOTE] 
   > In a production add-in, you would not want to use the same icon for two different buttons; but to simplify this tutorial, we'll do that. So the `Icon` markup in our new `Control` is just a copy of the `Icon` element from the existing `Control`. 

6. The `Action` element inside the original `Control` element has its type set to `ShowTaskpane`, but our new button isn't going to open a task pane; it's going to run a custom function that you create in a later step. So, replace `TODO5` with `ExecuteFunction`, which is the action type for buttons that trigger custom functions. The opening tag for the `Action` element should look like this:
 
    ```xml
    <Action xsi:type="ExecuteFunction">
    ```

> 7. The original `Action` element has child elements that specify a task pane ID and a URL of the page that should be opened in the task pane. But an `Action` element of the `ExecuteFunction` type has a single child element that names the function the control executes. You'll create that function in a later step, and it will be called `toggleProtection`. So, replace `TODO6` with the following markup:

Reviewer: Typically, we leave in the implicit "that"s. It helps with localization and readability. My freshman English teacher is cringing at me for suggesting this...

Author: ```suggestion
7. The original `Action` element has child elements that specify a task pane ID and a URL of the page that should be opened in the task pane. But an `Action` element of the `ExecuteFunction` type has a single child element that names the function that the control executes. You'll create that function in a later step, and it will be called `toggleProtection`. So, replace `TODO6` with the following markup:
```

Author: I understand the reasoning. Thanks!

1. Close all Office applications, including Excel. 

> 2. Delete the Office cache by deleting the contents (which may include files and subfolders) of the cache folder. This is necessary to completely clear the old version of the add-in from the host.

Reviewer: Consider making this more a more concrete statement about what's in the cache folder:

```suggestion
2. Delete the Office cache by deleting the contents (all the files and subfolders) of the cache folder. This is necessary to completely clear the old version of the add-in from the host.
```

3. Add the following markup to **popup.html**. Note:

>    - The page has an `<input>` field where the user will enter their name, and a button that will send this name to the task pane page where it will display.

Reviewer: I think "task pane" (not "task pane page") is more accurate, since it's embedded in Excel.
```suggestion
   - The page has an `<input>` field where the user will enter their name, and a button that will send this name to the task pane where it will display.
```
